### PR TITLE
[docs] feat: add agent loop debug-only quick start guide

### PR DESCRIPTION
**Pull request by Hailey-Zh**
### What does this PR do?

  Adds a step-by-step quick start guide (`docs/quick_start.md`) for running
  the uni-agent agent loop with minimal configuration: host deployment +
  dummy data + dummy reward using Qwen3-0.6B.

  ### Checklist Before Starting

  - [ ] Search for similar PRs or issues and paste at least one relevant link here: N/A
  (new doc)
  - [x] Format the PR title as `[docs] feat: add agent loop quick start guide`

  ### Test

  Manually verified the full pipeline end-to-end with Qwen3-0.6B on NPU:
  data → agent loop → trajectory → reward → GRPO training.

  ### API and Usage Example

  No API changes.

  ### Design & Code Changes

  Single-file addition. The guide covers:
  - clone & install
  - model download + chat template replacement
  - dummy reward, agent config, training data, and training script creation
  - launch and verification
  - common pitfalls (prompt format, chat template mismatch)

  ### Checklist Before Submitting

  - [x] Read the [Contribute Guide](../CONTRIBUTING.md)
  - [x] Confirm the PR title matches the required format
  - [x] Confirm the placeholder text in this template has been replaced with real content

### CODE REVIEW

**Comment by tardis-key** [comment]
move dummy reward to code

**Comment by tardis-key** [comment]
add QuickStart about Gateway

**Comment by Hailey-Zh** [comment]
> move dummy reward to code

Done. Added dummy.py to uni_agent/reward/ and registered it.

File: docs/quick_start.md [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+
+## 2. Clone & Install
+
+```bash
+git clone https://github.com/verl-project/uni-agent.git ~/uni-agent
+cd ~/uni-agent
+
+# verl submodule — provides the GRPO training engine
+git submodule update --init --recursive
+pip install --no-deps -e ./verl
```
Comment (tardis-key) [inline docs/quick_start.md]: Already requires a working verl GRPO environment as a prerequisite — why does it still need to install verl? There's a logical inconsistency.

File: docs/quick_start.md [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+# Other dependencies
+pip install swe-rex loguru pydantic pydantic_settings aiohttp
+```
+
+## 3. Download Model & Replace Chat Template
+
+### 3.1 Download the model
+
+```bash
+modelscope download Qwen/Qwen3-0.6B --local-dir $HOME/models/Qwen3-0.6B
```
Comment (tardis-key) [inline docs/quick_start.md]: Same as above.

File: docs/quick_start.md [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+```
+
+
+## 4. Create Dummy Reward
+
+Create `~/uni-agent/uni_agent/reward/dummy.py`:
```
Comment (tardis-key) [inline docs/quick_start.md]: If the code already includes it, remove it.

File: docs/quick_start.md [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+  tools:
+    - name: execute_bash
+    - name: submit
+  reward:
+    name: dummy
+```
+
+## 6. Create Dummy Training Data
+
+Create `generate_dummy_data.py`:
```
Comment (tardis-key) [inline docs/quick_start.md]: already done in code
